saleor/checkout/migrations/0041_auto_20220329_1612.py

```diff
@@ -0,0 +1,157 @@
+# Generated by Django 3.2.12 on 2022-03-29 16:12
+
+from decimal import Decimal
+
+import django.utils.timezone
+from django.db import migrations, models
+
+
+def set_default_checkout_line_currency(apps, schema_editor):
+    CheckoutLine = apps.get_model("checkout", "CheckoutLine")
+    lines = CheckoutLine.objects.select_related("checkout")
+
+    for line in lines:
+        line.currency = line.checkout.currency
+
+    CheckoutLine.objects.bulk_update(lines, ["currency"])
```

Comment:
@fowczarek @cmiacz We'd like to hear your opinion on performance of this piece.

Reply:
IMO there is a risk of OOM  here.  Also not sure if we shouldn't have a separate migration for populating this field